src/match-files.ts

```diff
@@ -0,0 +1,573 @@
+/// <reference path="../typings/node/node.d.ts"/>
+/// <reference path="../typings/typescript/typescript.d.ts"/>
+
```

Comment:
Add a comment indicating much of this file was initially copied over from `src/compiler/sys.ts` and `src/compiler/core.ts`.